code/game/machinery/porta_turret/portable_turret.dm

```diff
@@ -592,6 +592,29 @@
 	cover.name = name
 	cover.desc = desc
 
+/obj/machinery/porta_turret/centcomm_shuttle

```

Comment:
Just looked at it and assess_perp has to return 0 to work properly, if i make it a subtype of the syndicate turrets the return value will be 10 and i am pretty sure that breaks what it shoots at if from prior testing.